mentoring/tests/integration/test_data_export.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
```

Comment:
@antoviaque Oh, well if we don't need data-export then there's no compelling reason to keep the django database tables and models around, if I recall. We could use a pure XBlock field approach for storing the answers as in 7eadc3f89a5411070d5a19430025c355f1d89f6b.